case-studies/aepsi.html

```diff
@@ -0,0 +1,123 @@
+      width="100%">
+    <figcaption>AEPSi Ready Set Assessment</figcaption>
+  </figure>
+
+  <p>Leveraging technologies like <a target="_blank"href="https://smartlogic.io/technologies/elixir-phoenix">Elixir, Phoenix LiveView</a>, PostgreSQL, and Tailwind CSS, SmartLogic built a responsive, real-time application that ensures seamless interactivity and data management. The use of Oban for background job processing and custom PDF generation tools allows for efficient data reporting and comprehensive, printable assessments.</p>
+
+</div>
+</section>
+
+<div class="content-page-wrapper">
+    <section class="section content-block">
+      <div class="container">
+
+        <h2>Enhancing Data Collection with Real-Time Interactivity</h2>
+
+        <p>The AEPSi platform incorporates dynamic features like interactive assessments and customizable reporting to enhance data collection efficiency and provide users with real-time feedback. Educators can instantly view and record developmental progress, ensuring that no small increment of growth goes unnoticed.</p>
+
+        <p>By integrating real-time interactivity and <a target="_blank"href="https://smartlogic.io/podcast/elixir-wizards/s12-e03-background-jobs-web-development/">background job processing</a> through Phoenix LiveView and Oban, the platform offers a seamless and responsive user experience. This innovative approach streamlines data management and empowers educators and caregivers with immediate, actionable insights.</p>
+
+        <h2 style="margin-top: 20px;">Advanced Reporting Capabilities</h2>
+
+        <p>To support educators and caregivers in their mission to track and nurture child development, the AEPSi platform includes advanced reporting capabilities. One of the standout features is the child progress record, which is visually engaging and rich with graphical elements such as shapes and arrows. This report provides a clear, comprehensive view of each child's developmental progress, making it easier for educators to identify areas of improvement and success.</p>
+
+        <figure>
+            <img src="{{ site.baseurl }}/images/case-studies/aepsi/AEPS3ChildProgressReportv3.gif" alt="AEPSi Child Progress Record"
+              width="100%">
+            <figcaption>AEPSi Child Progress Record</figcaption>
+          </figure>
+
+          <h2 style="margin-top: 20px;">App Design and Development Highlights </h2>
+
+          <ul>
+            <li><b>Custom PDF Generation:</b> SmartLogic utilized Phoenix components for <a target="_blank"href="https://smartlogic.io/podcast/elixir-wizards/s12-e07-web-to-pdf-printing-solutions/">creating PDF layouts.</a> This system takes child meta information and applies headers, footers, and logos to generate an HTML document. The document is then processed through a backend tool called Prince (CLI), converting it into a PDF with custom styles, headers, and footers on every page.</li>
```

Comment:
Hope this clarifies things a bit:

```suggestion
            <li><b>Custom PDF Generation:</b> SmartLogic utilized Phoenix components for <a target="_blank"href="https://smartlogic.io/podcast/elixir-wizards/s12-e07-web-to-pdf-printing-solutions/">creating PDF layouts.</a> This system embeds report and assessment content in the main section, with repeating headers containing account, program, or child meta information and footers containing credit text. A backend tool called Prince (CLI) converts this layout HTML into a PDF file with appropriate CSS printing styles.</li>
```